system-tests/README.md

```diff
@@ -1,62 +1,106 @@
 ## System tests
 
-The test uses the key vault secret to connect to the storage accounts and copy a file from provider to consumer storage account.
+The test copy a file from provider to consumer blob storage account.
 
 ### Building MVD project
 
-One of the MVD dependencies is the Registration Service REST client library. Registration Service is not published to any central artifactory yet so in local 
-development we have to use locally published dependencies.
+MVD dependencies are Eclipse DataSpaceConnector(EDC) and Registration Service. Both of these dependencies are not published to any central artifactory yet so in local
+development we have to use locally published dependencies, once this is done MVD can be build using
 
-#### Publish Registration Service to local Maven
+```bash
+./gradlew build
```

Comment:
As discussed - it would make sense to exclude the test task here, otherwise the CatalogClientTest fails

Reply:
Updated. Thank you.